eu.stethewwolf.gresistor.json

```diff
@@ -0,0 +1,30 @@
+{
+    "app-id": "eu.stethewwolf.gresistor",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "gresistor3",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules": [
+        {
+            "name": "gresistor",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST}",
+                "install -Dm644 gresistor.desktop  -t ${FLATPAK_DEST}/share/applications",
+                "install -Dm644 gresistor3/pixmaps/gresistor.png  -t ${FLATPAK_DEST}/share/icons"
```

Comment:
this is not the right path for application icons.

`${FLATPAK_DEST}/share/icons/hicolor/SIZE/apps/`

where `SIZE` is the like 128x128 for a 128 pixels icon.
(that's the only required size)

Reply:
and probably need a `"rename-icon": "gresistor",` also above line 7

Reply:
looking at it, it seems that upstream icon is only 48x48.